dev/releases/update_website.py

```diff
@@ -65,40 +70,47 @@
 # (global variable <release>, with assets <assets>) to <writedir>.
 def download_asset_by_name(asset_name, writedir):
     try:
-        url = [ x for x in assets if x.name == asset_name ][0].browser_download_url
+        url = [x for x in assets if x.name == asset_name][0].browser_download_url
     except:
-        error(f"Cannot find {asset_name} in the GitHub release with tag {release.tag_name}")
+        error(
+            f"Cannot find {asset_name} in the GitHub release with tag {release.tag_name}"
+        )
 
     with utils.working_directory(writedir):
         notice(f"Downloading {url} to {writedir} . . .")
         utils.download_with_sha256(url, asset_name)
 
-def extract_tarball(tarball):
-    notice(f"Extracting {tarball} . . .")
-    with tarfile.open(tarball) as tar:
+
+def extract_tball(tball):
```

Comment:
```suggestion
def extract_tarball(tarball):
```
and then also change `tball` below...